student-bios/Alexander-Segovia.md

```diff
@@ -0,0 +1,26 @@
+# Introducing myself
+
+<img src="https://i.redd.it/a2gfj1ookcm81.jpg" width="200" height="200" alt="my first pokemon">
+
+A very motivated student on his way to becoming an impressive future programmer.
+I am 29 years old and come from El Salvador, Central America. My mother tongue
+is Spanish and I also speak English and a little bit of Flemish. I moved to
+Belgium on 2021 and I am really happy to start a coding career here.
+
+**Some facts about me:**
+
+✈️ Traveling is one of my favorite things in life
+
+🏢 I like museums
+
+🏃 Fitness life and healthy food are topics I am interested in
+
+🐉 Pokemon Go is the game that drives me crazy!
+
+🙇 I am an introverted person, but not an antisocial one at all!
```

Comment:
If you want, you can also use the :emoji-name: written version of emojis, which you can find here:
[Complete list of github markdown emoji markup - gists · GitHub](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwiGvrDx08H2AhWVh_0HHaqCCpIQFnoECAQQAQ&url=https%3A%2F%2Fgist.github.com%2Frxaviers%2F7360908&usg=AOvVaw3GUfukqzTmILotl0Gi-xDe)